test/client/jest.config.json

```diff
@@ -1,9 +1,16 @@
 {
+	"moduleNameMapper": {
```

Comment:
What is this useful for?

Reply:
You can provide a different implementation for a file. In this case our client config reads data from global variable exposed by the server in HTML script tag. That's why we need to provide a workaround :)

Reply:
To make I understood: does it mean that when in a client test, any call to config will be magically using this?

Reply:
Yes, this is how it works. It is not as flexible as Webpack, but provides the similar capabilities. You can use regular expressions to match files.